daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -275,9 +275,17 @@ class RazerOrbweaverChroma(_RazerDeviceBrightnessSuspend):
     USB_VID = 0x1532
     USB_PID = 0x0207
     DEDICATED_MACRO_KEYS = True
-    METHODS = ['get_device_type_keypad', 'set_breath_random_effect', 'set_breath_single_effect',
-               'set_breath_dual_effect', 'set_static_effect', 'set_spectrum_effect', 'set_wave_effect', 'keypad_get_profile_led_red', 'keypad_set_profile_led_red', 'keypad_get_profile_led_green',
-               'keypad_set_profile_led_green', 'keypad_get_profile_led_blue', 'keypad_set_profile_led_blue', 'get_macros', 'delete_macro', 'add_macro', 'keypad_get_mode_modifier', 'keypad_set_mode_modifier']
+    HAS_MATRIX = True
+    MATRIX_DIMS = [5, 23]
```

Comment:
`[4, 22]` would be the correct values for your device I think

Reply:
That's also what I thought and tried first. But then I could not access the fourth row. No idea why. But I'll revalidate and be back. Maybe I messed something up then.

Reply:
Using [4, 22] leads to the following error ([5, 23] works fine):
device.fx.advanced.matrix[4, 1] = (255, 255, 0)
  File "/usr/lib/python3/dist-packages/openrazer/client/fx.py", line 988, in __setitem__
    assert 0 <= key[0] < self._rows, "Row out of bounds"

Reply:
Sorry my bad. For some reason the 0th row isn't used on this device it seems. Then the matrix_dims should be `[5, 22]`. Row `1` to Row `4` and Column `0` to Column `21` are being sent by Synapse